### PR TITLE
PR feature/08--surf+paddle+rwd

### PR DESCRIPTION
**Pull request by mfees**
https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/109

### CODE REVIEW

**Comment by mfees** [comment]
http://htmlpreview.github.io/?https://github.com/mfees/TIY-Assignments/blob/feature/08--surf%2Bpaddle%2Brwd/pub/index.html

**Comment by jrutishauser** [comment]
:+1:

**Comment by jaredstevick** [comment]
:+1: the site is looking really good!

**Comment by al-the-x** [comment]
Something I noticed from last time that I forgot to mention popped up again. In your Skitch specs (which are great, BTW), you identify the pull quote as an `<h3>` with `border-left`... That should be a `<blockquote>` tag, not a heading.

**Comment by al-the-x** [comment]
Additionally, I would wrap the headline, byline and publication date of the article inside a `<header>` tag instead of a `<section>` and the sharing buttons inside a `<footer>` tag instead of a generic `<div>`... but you know how I feel about that tag. Reason being, those parts are likely repeated on _every_ article, making them the "header" and "footer" for the article. No rule saying you can't reuse those semantic tags inside a different context, i.e. you can have a page-level header _and_ an article-level header _and sidebar header_ for that matter. Madness!

**Comment by al-the-x** [comment]
I can also tell that you used a different HTML5 Boilerplate package than the one that I provided. Nothing wrong with that... Motivation?

File: pub/index.html [hunk 1]
```diff
@@ -4,40 +4,154 @@
...
+        <div class="main-container">
+            <section class="content">
+                <div class="text-container">
+                    <h2>I like big cats and i can not lie</h2>
+                    <time datetime="2015-07-09T19:22Z-4">
+                    <figure class="aside-image">
+                        <img src="http://lorempixel.com/output/cats-q-c-100-100-7.jpg">
+                        <figcaption>Cattie Cat</figcaption>
+                    </figure>   
+                    <small>on Monday, June 7th 9:22pm</small>
```
Comment (al-the-x) [inline pub/index.html:49]: This should really be the only contents of the `<time>` tag... And I've created a monster by showing you that `<figure>` tag. Girl, you're in _love_! 

While there's nothing wrong with your usage, better to provide the author information within `<address class="author">` or `<a href rel="author">` that links to a user profile, etc.

File: pub/index.html [hunk 2]
```diff
@@ -4,40 +4,154 @@
...
+                </li>
+            </ul>    
+            <div class="button-group">
+                <icon class="fa fa-comment-o fa-lg"></icon>
+                <div class="button-text hidden-mobile">Read Comments</div>
+                <icon class="fa fa-cloud-upload fa-lg"></icon>
+                <div class="button-text hidden-mobile">Read Later</div>
+                <icon class="fa fa-share-square-o fa-lg"></icon>
+                <div class="button-text hidden-mobile">Share This Post</div>
+            </div>
```
Comment (al-the-x) [inline pub/index.html:75]: Swimming slightly ahead of the pack, I see. Hopefully you can see the semantic value of those Bootstrap classes.

File: pub/index.html [hunk 3]
```diff
@@ -4,40 +4,154 @@
...
+                <a href="#">Nav Item 1  |</a>
+                <a href="#">Nav Item 2  |</a>
+                <a href="#">Nav Item 3  |</a>
+                <a href="#">Nav Item 4  |</a>
+                <a href="#">Nav Item 5</a>
+            </nav>
+            <div class="copyright-content">
+                <img src="img/tiy-logo.png">
+                <p>The Ironyard Copyright 2013 All Rights Reserved</p>
+            </div>
```
Comment (al-the-x) [inline pub/index.html:139]: Good choice putting the footer nav _above_ the copyright block in your HTML. Even though it appears to the _right_ in the full-sized view, the nav appears _before_ the copyright in mobile.

File: pub/css/main.css [hunk 4]
```diff
@@ -198,39 +402,319 @@ textarea {
...
+   These examples override the primary ('mobile first') styles.
+   Modify as content requires.
+   ========================================================================== */
+@media only screen and (max-width: 678px) {
+    .hidden-mobile {
+        display: none;
+    }
+}
+
+@media only screen and (min-width: 678px) {
```
Comment (al-the-x) [inline pub/css/main.css:455]: Wrapping the _entire_ stylesheet in a media query is a bit of overkill. With a _whatever_-first approach in mind, the default styles in your CSS file should be geared towards that _whatever_ -- mobile, tablet or desktop -- and unburdened by a resolution selector. The MQs come into play by _overriding those defaults_ via the cascading rules when the viewport or device size matches the query. I hope the discussion today shed some light on that part of RWD for you... The way MQs work with the cascade is fundamental to building efficient responsive layouts.

**Comment by al-the-x** [comment]
You may be overusing CSS classes, which is consistent with your experience level in CSS. In order to advance, carefully consider your use of classes and reflect on the use of contextual selectors to replace them.

**Pull request merged**